util/devel/check_paths

```diff
@@ -39,9 +39,9 @@ def check_path(p):
     """returns True if p exists, False if it doesn't"""
 
     # These are common written-out alternatives, not paths
-    if p == "compiler/runtime":
-        return True
-    if p == "compiler/module":
+    if (p == "compiler/runtime" or p == "compiler/module" or
+        p == "runtime/library" or p == "runtime/lib" or
+        p == "module/files" or p == "test/directory" ):
```

Comment:
Think we want "modules/files" (plural "modules"), at least if that string is for this one:
```
doc/rst/developer/chips/inactive/21.rst:74: modules/files not found
```